src/components/AppCatalog/AppDetail/InstallAppForm.js

```diff
@@ -107,43 +106,37 @@ const InstallAppForm = ({
 
       {formAbilities.hasFixedNamespace ? (
         <Input
-          key='fixed-namespace'
           description={`This app must be installed in the ${formAbilities.fixedNamespace} namespace`}
-          hint={<>&nbsp;</>}
+          key='fixed-namespace'
           label='Namespace:'
-          value={formAbilities.fixedNamespace}
           readOnly={true}
+          value={formAbilities.fixedNamespace}
         />
       ) : (
         <Input
-          key='dedicated-namespace'
           description='We recommend that you create a dedicated namespace. The namespace will be created if it doesn’t exist yet.'
-          hint={<>&nbsp;</>}
+          key='dedicated-namespace'
```

Comment:
I am unsure why we need the `key` prop

Reply:
Not sure if this is that case, but I remember we had a problem at some point, where React would believe that it was the same element and it would not re-render it, because it thought it didn't change, and we could force a re-render by using different keys

Reply:
Ok, I'll leave them alone :+1: